### PR TITLE
Deploy multinode env

### PR DESCRIPTION
**Pull request by jackhodgkiss**

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/inventory/hosts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# A single all-in-one controller/compute host.
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/inventory/hosts]: Remove me :)

File: etc/kayobe/environments/ci-multinode/stackhpc-ci.yml [hunk 2]
```diff
@@ -48,5 +48,11 @@ stackhpc_repo_docker_ce_ubuntu_version: "{{ stackhpc_pulp_repo_docker_ce_ubuntu_
 stackhpc_docker_registry: "{{ stackhpc_repo_mirror_url | regex_replace('^https?://', '') }}"
 
 # Username and password of container registry.
-stackhpc_docker_registry_username: ""
-stackhpc_docker_registry_password: ""
+stackhpc_docker_registry_username: "stackhpc-kayobe-ci"
+stackhpc_docker_registry_password: !vault |
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/stackhpc-ci.yml:52]: Did you create a new vault password for this?

Comment (jackhodgkiss) [reply etc/kayobe/environments/ci-multinode/stackhpc-ci.yml:52]: I used the one you provided. However, I think there was some confusion at the start of this project regarding the password.

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/stackhpc-ci.yml:52]: I don't mean the registry password, but the ansible vault password used to encrypt it here

File: etc/kayobe/environments/ci-multinode/inventory/hosts (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/inventory/hosts]: Sorry, I meant remove the entire file, and the other files that were removed in the previous PR

File: etc/kayobe/environments/ci-multinode/kolla/passwords.yml [hunk 3]
```diff
@@ -0,0 +1,1610 @@
+$ANSIBLE_VAULT;1.1;AES256
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/kolla/passwords.yml]: Still need to remove this one

**Review by markgoddard — APPROVED** [review]

**Pull request merged**